easybuild/easyconfigs/r/R-bundle/pbdBASE-02.3-0_ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,45 @@
+# Add support for intel compilers by removing references to BI_COMM_WORLD
```

Comment:
I notified the pbd people about the patch and I'll see if I can help land it on their system. They seem willing to accept contributions but they don't have an `icc/impi` system so that's like the only reason they may be reluctant.